src/components/application_manager/test/resumption/resume_ctrl_test.cc

```diff
@@ -152,6 +152,8 @@ TEST_F(ResumeCtrlTest, StartResumption_AppWithGrammarId) {
       .WillRepeatedly(DoAll(SetArgReferee<2>(saved_app), Return(true)));
   EXPECT_CALL(*app_mock, UpdateHash());
   EXPECT_CALL(*app_mock, set_grammar_id(kTestGrammarId_));
+  EXPECT_CALL(state_controller_, SetRegularState(_, kDefaultTestLevel_))
+      .Times(1);
```

Comment:
@vlantonov `Times(1)` is redundant. Of course it can be kept but to keep consistency - we have 2 ways: Add to all EXPECT_CALL (when the method is called just once) or use default value (which is 1)